File: DSA/Manav-Rajput/Day26_question1.java
class Solution {
    public boolean isPalindrome(int x) {
        int n=x;
        int c=0;
        if(x>=0)
        {
            while(n!=0)
            {
                int digit=n%10;
                c=c*10+digit;
                n=n/10;
            }
            if(c==x)
            {
                return true;
            }
            else{
                return false;
            }
        }
        else{
            return false;
        }
    }
}

File: DSA/Arshdeep-Kaur/Day 26/Day26_Question1.cpp
class Solution {
public:
    bool isPalindrome(int x) {
        string str = to_string(x);
        int i = 0, j = str.length()-1;

        while(i <= j) {
            if (str[i] == str[j]) {
                i++;
                j--;
            }
            else {
                return false;
            }
        }
        return true;
    }
};
